src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -758,10 +758,10 @@ public function simplePaginate($perPage = null, $columns = ['*'], $pageName = 'p
     /**
      * Save a new model and return the instance.
      *
-     * @param  array  $attributes
+     * @param  array|Fillable  $attributes
      * @return \Illuminate\Database\Eloquent\Model|$this
      */
-    public function create(array $attributes = [])
+    public function create($attributes = [])
```

Comment:
:blush: I like the idea, but I would dare to bring it a bit forward. 

We could break this here and just accept the `validated` objects. By doing so, we completely remove the risk of mass assignment since those objects/interfaces can expose a `toArray` method to serialize the final feeding.

The same would apply to all the array feeding methods we have across eloquent. Therefore, eloquent can expose the interface to be implemented.

:neckbeard: If gone this route, we could even have specialized objects for each database update action, such as: insert, update, delete, etc. 

:boom: Nice work.